starcheck/calc_ccd_temps.py

```diff
@@ -152,8 +152,19 @@ def get_ccd_temps(oflsdir, outdir='out',
 
     # Get tstart, tstop, commands from backstop file in opt.oflsdir
     bs_cmds = get_bs_cmds(oflsdir)
-    tstart = DateTime(bs_cmds[0]['date']).secs
-    tstop = DateTime(bs_cmds[-1]['date']).secs
+
+    # Use RLTT and SCHEDULED_STOP_TIME if available
+    rltt = bs_cmds['event_type'] == 'RUNNING_LOAD_TERMINATION_TIME'
+    if np.any(rltt):
+        tstart = DateTime(bs_cmds['date'][rltt][0]).secs
+    else:
+        tstart = DateTime(bs_cmds[0]['date']).secs
```

Comment:
Make this consistent with the line above with `bs_cmds['date'][0]`.  This brings it in lines with current best practice of always selecting the column first, then the item within the column.  This is much faster.  Yes, it is changing code, but in a way that is safe and leaves this code in a better way.